dbt/include/spark/macros/adapters.sql

```diff
@@ -229,9 +229,38 @@
   {% endfor %}
 {% endmacro %}
 
+{% macro get_column_comment_sql(column_name, column_dict) -%}
+  {% if (column_name|upper in column_dict) -%}
+    {% set matched_column = column_name|upper -%}
+  {% elif (column_name|lower in column_dict) -%}
+    {% set matched_column = column_name|lower -%}
+  {% elif (column_name in column_dict) -%}
+    {% set matched_column = column_name -%}
+  {% else -%}
+    {% set matched_column = None -%}
+  {% endif -%}
+  {% if matched_column and column_dict[matched_column]["description"] -%}
+    {% set column_comment_clause = "comment '" ~ column_dict[matched_column]["description"] ~ "'" %}
+  {%- endif -%}
+  {{ adapter.quote(column_name) }} {{ column_comment_clause }}
+{% endmacro %}
+
+{% macro get_persist_docs_column_list(model_columns, query_columns) %}
+(
```

Comment:
Can we move the parenthesis to the calling macro? I don't think this would be reused often, but if it does get reused, it's preferable to leave something like that as an option in the calling macro versus requiring it here.

Reply:
moved